xbmc/cores/VideoPlayer/VideoRenderers/BaseRenderer.cpp

```diff
@@ -190,10 +196,26 @@ void CBaseRenderer::CalcNormalRenderRect(float offsetX, float offsetY, float wid
   else if (verticalShift < -1.0f)
     posY += shiftRange * (verticalShift + 1.0f);
 
-  m_destRect.x1 = (float)MathUtils::round_int(static_cast<double>(posX + offsetX));
-  m_destRect.x2 = m_destRect.x1 + MathUtils::round_int(static_cast<double>(newWidth));
-  m_destRect.y1 = (float)MathUtils::round_int(static_cast<double>(posY + offsetY));
-  m_destRect.y2 = m_destRect.y1 + MathUtils::round_int(static_cast<double>(newHeight));
+  destRect.x1 = (float)MathUtils::round_int(static_cast<double>(posX + offsetX));
```

Comment:
While you are at it, mind changing the c-style casts to a static_cast ?

Reply:
sure, just pushed an update